mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensor.h

```diff
@@ -173,6 +173,14 @@ AffineMap inferLvlToDim(AffineMap dimToLvl, MLIRContext *context);
 /// Asserts on failure (so only use when known to succeed).
 AffineMap inverseBlockSparsity(AffineMap dimToLvl, MLIRContext *context);
 
+/// Given the dimToLvl map, returns the block size in a vector.
+/// For instance, a 2x3 block will return [2, 3].
```

Comment:
perhaps also document that unblocked dimensions are 0 (or perhaps 1, but we should reserve that for explicit "blocking" by 1. So the length of the vector is DimRank and we have 0 is that dimension is not blocked or != if blocked.